README.md

```diff
@@ -80,49 +99,61 @@ models:
     quickbooks_source:
         +schema: my_new_schema_name
 ```
-### Disabling models
 
-This package takes into consideration that not every QuickBooks account utilizes the same transactional tables, and allows you to disable the corresponding functionality. By default, most variables' values are assumed to be `true` (with exception of purchase orders and credit card payments). Add variables for only the tables you want to disable or enable respectively:
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_quickbooks_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
+    
+```yml
+vars:
+    quickbooks_<default_source_table_name>_identifier: your_table_name 
+```
+
+### Unioning Multiple QuickBooks Connectors 
```

Comment:
Very minor and my appologies for not raising this at the start. I feel this is a more important additional configuration than the others within this collapsable section. Can you make this the first section in the `(Optional) Step 5: Additional Configurations` section.

My reasoning here is that I would prefer customers see this additional config first (since it is probably most useful to them) rather than the schema and source changes which are more edge cases.

Reply:
Ah good call! Flipped it.